package.json

```diff
@@ -7,7 +7,7 @@
     "create-data": "netlify dev:exec node scripts/create-example-data.js",
     "bootstrap": "yarn create-db && yarn create-data",
     "build": "parcel build src/index.html",
-    "dev": "parcel src/index.html",
+    "dev": "parcel src/index.html 1> /dev/null",
```

Comment:
I get an error I think because of this:

```
> netlify-faunadb-graphql-auth@0.1.1 dev
> parcel src/index.html 1> /dev/null

The system cannot find the path specified.
```

Reply:
This now uses Parcel's `--log-level` option.

Reply:
It would make sense to me, but don't know how to make it stop.  The default netlify dev experience came with it and I am not sure how to turn it off 🤷

Reply:
`parcel watch` will basically do `parcel [file]` (which implies `parcel serve`) but without the server - except `netlify dev` doesn't spin up the functions after... We can probably just leave it the way it is tbh.